frontend/src/app/modules/bim/ifc_models/toolbar/import-export-bcf/bcf-import-button.component.ts

```diff
@@ -33,7 +33,9 @@ import {BcfPathHelperService} from "core-app/modules/bim/bcf/helper/bcf-path-hel
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/dd7f161bb9bbc802f64ff43ae6b91ecdc17e5161/frontend/src/app/modules/bim/ifc_models/toolbar/import-export-bcf/bcf-import-button.component.ts#L33"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>